examples/purchaseTester/yarn.lock

```diff
@@ -5926,15 +5926,15 @@ react-native-web@^0.11.7:
     prop-types "^15.6.0"
     react-timer-mixin "^0.13.4"
 
-react-native@^0.64.0:
-  version "0.64.0"
-  resolved "https://registry.yarnpkg.com/react-native/-/react-native-0.64.0.tgz#c3bde5b638bf8bcf12bae6e094930d39cb942ab7"
-  integrity sha512-8dhSHBthgGwAjU+OjqUEA49229ThPMQH7URH0u8L0xoQFCnZO2sZ9Wc6KcbxI0x9KSmjCMFFZqRe3w3QsRv64g==
+react-native@^0.64.1:
```

Comment:
is it weird that this line and the next don't match?

Reply:
Nah, it's ok, the caret operator will match until the next major. If the dependency were ^0.64.0, we would also be using 0.64.2, we probably just never ran it when 0.64.1 was released